aws-sqs-queue/queues.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_sqs_queue" "normal" {
+  name                      = "${var.service_name}_Queue${var.fifo_queue ? ".fifo" : ""}"
```

Comment:
Hmm, všechny servisy se toho drží? A není někde těch front víc? Přemýšlím jestli přidat variable `name` a tohle nechat jako default název a nebo to vyplňovat pokaždý. 🤔

Reply:
No ten `service_name` je spis nastrel a nejradsi bych to pojmenoval jako `prefix` - prislo by mi fajn vynutit ten konec stejnej takhle.

Aaa jeste jsem se koukal a jediny OSB nepouziva `_Queue` a `_DeadLetterQueue` ale misto toho `_queue` a `_dlq` - neslo by to nejak prekopat, aby to pouzivalo to co ostatni servicy?

Reply:
Já spíš nechápu proč by tam vůbec nějaký `_Queue` suffixy jsou. Bych byl pro se jich zbavit. 🙂  Spíš bych u toho name jmennou konvenci nezaváděl, např. Message Router vytváří front několik: https://github.com/FigurePOS/fgr-message-router/tree/master/tf

Reply:
A DLQ pojmenovat jako "${var.name}_dlq"? (Čímž bychom museli smazat ty existující a vytvořit nový, přejmenovávat se nedají. U DLQ to asi neva, akorát se budou muset upravit Datadog alerty. Což mi připomíná, že každá DLQ fronta by měla mít DD alert a ten by mohl být zapečenej taky v tomhle modulu. 😁

Reply:
Na ty DD alerty už jeden modul je: https://github.com/FigurePOS/terraform-modules/blob/master/datadog-sqs/datadog_monitor.tf ale přemýšlím že by teda dost mohlo dávat smysl ho přimergnout sem. Předpokládám že u každý fronty chceme sledovat jestli se tam toho nehromadí moc a jestli neppřibývají zprávy do DLQ.